src/auth0/handlers/guardianPhoneFactorMessageTypes.js

```diff
@@ -41,7 +65,7 @@ export default class GuardianPhoneMessageTypesHandler extends DefaultHandler {
     const { guardianPhoneFactorMessageTypes } = assets;
 
     // Do nothing if not set
-    if (!guardianPhoneFactorMessageTypes) return;
+    if (!guardianPhoneFactorMessageTypes || !guardianPhoneFactorMessageTypes.message_types) return;
```

Comment:
This is to handle the empty object case (when the API does not support the endpoint).